src/babel-plugin-import-meta.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import importMetaSyntax from '@babel/plugin-syntax-import-meta';
+import template from '@babel/template';
```

Comment:
`import {ast} from '@babel/template'`?

Reply:
unfortunately, it's a default export of an object with an `ast` property.